How-to-Contribute.md

```diff
@@ -9,7 +9,7 @@ If you want to understand how Code works or want to debug an issue, you'll want
 
 ### Installing Prerequisites
 
-You'll need [Node.JS](https://nodejs.org/en/), at least `v5.10`.
+You'll need [git](https://git-scm.com) and [Node.JS](https://nodejs.org/en/), at least `v5.10`.
```

Comment:
I like it, can you reword to be more clear that the v5.10 is for node? Maybe "You'll need git and at least v5.10 Node.JS"